drracket/drracket/private/stick-figures.rkt

```diff
@@ -216,14 +211,14 @@
     (connect 'right-ankle 'right-toe points dc factor dx dy))
   
   (define (connect from to points dc factor dx dy)
-    (let ([from-p (assoc from points)]
-          [to-p (assoc to points)])
-      (when (and from-p to-p)
-        (send dc draw-line 
-              (+ dx (* factor (list-ref from-p 1)))
-              (+ dy (* factor (list-ref from-p 2)))
-              (+ dx (* factor (list-ref to-p 1)))
-              (+ dy (* factor (list-ref to-p 2)))))))
+  (define from-p (assoc from points))
```

Comment:
This line doesn't look indented inside `connect`.

Reply:
Fixed